packages/cactus-test-cmd-api-server/src/test/typescript/integration/runtime-plugin-imports.test.ts

```diff
@@ -32,26 +32,25 @@ test("can import plugins at runtime (CLI)", async (t: Test) => {
   apiServerOptions.cockpitPort = 0;
   apiServerOptions.grpcPort = 0;
   apiServerOptions.apiTlsEnabled = false;
-  apiServerOptions.plugins = [
-    {
-      packageName: "@hyperledger/cactus-plugin-keychain-memory",
-      type: PluginImportType.Local,
-      options: {
-        instanceId: uuidv4(),
-        keychainId: uuidv4(),
-        logLevel,
-      },
-    },
-  ];
   const config = configService.newExampleConfigConvict(apiServerOptions);
 
   const apiServer = new ApiServer({
     config: config.getProperties(),
   });
+  afterAll(() => apiServer.shutdown());
 
-  await t.doesNotReject(
-    apiServer.start(),
-    "failed to start API server with dynamic plugin imports configured for it...",
-  );
-  test.onFinish(() => apiServer.shutdown());
+  test(testCase, async () => {
+    apiServerOptions.plugins = [
+      {
+        packageName: "@hyperledger/cactus-plugin-keychain-memory",
+        type: PluginImportType.Local,
+        options: {
+          instanceId: uuidv4(),
+          keychainId: uuidv4(),
+          logLevel,
+        },
+      },
+    ];
+    await expect(apiServer.start()).toBeTruthy();
```

Comment:
@awadhana Truthiness of the returned promise does not imply that it will be resolved (e.g. not rejected)

Reply:
Changes have been made for this

Reply:
@TonyRowntree You are accessing the function reference instead of calling the function so it will resolve instantly always instead of doing the assertion itself.